DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/SlewingCoef.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FT0_SLEWINGCOEF_H_
+#define ALICEO2_FT0_SLEWINGCOEF_H_
+////////////////////////////////////////////////
+// Slewing coefficients for FT0
+//////////////////////////////////////////////
```

Comment:
Nit-picky and of course not blocking but maybe use the commenting style from the guidelines: [https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html#Files](https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html#Files)?

Reply:
Ah, yeah, we have it also here https://github.com/AliceO2Group/AliceO2/blob/ebd3382aca03d8945bae5005d24ff991ba880bba/DataFormats/Detectors/FIT/common/include/DataFormatsFIT/LookUpTable.h#L14-L16 I guess later we can apply cosmetics